templates/catalog/_partials/product-add-to-cart.tpl

```diff
@@ -6,19 +6,40 @@
   {if !$configuration.is_catalog}
     <div class="mb-3">
       {block name='product_availability'}
-        <span id="product-availability" class="product__availability js-product-availability d-flex align-items-center">
```

Comment:
With all the logic out here this could be separated into another file, what do you think

Reply:
I would do it properly when somebody remakes all this `page-refresh-cant-move-anything-crap` :-)